posthog/models/product_intent/product_intent.py

```diff
@@ -106,6 +106,9 @@ def has_activated_error_tracking(self) -> bool:
         # the team has resolved any issues
         return ErrorTrackingIssue.objects.filter(team=self.team, status=ErrorTrackingIssue.Status.RESOLVED).exists()
 
+    def has_activated_surveys(self) -> bool:
+        return Survey.objects.filter(team=self.team, start_date__isnull=False).exists()
```

Comment:
I think we should use `team__project_id=team.project_id` here instead similar to https://github.com/PostHog/posthog/pull/30631/files